alucio/hooks/usePlayerEvent/usePlayerEvent.tsx

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useState } from 'react';
+
+export default function usePlayerEvent() {
+  const [msg, setMsg] = useState<MessageEvent>()
+
+  const handleMsgRec = ($e) => {
+    // TODO: Add message processing if necessary
+    // console.log('got event from player', $e)
+    setMsg($e)
+  }
+
+  useEffect(() => {
+    window.addEventListener('message', handleMsgRec)
+    const cb = () => {
+      window.addEventListener('message', handleMsgRec)
+    }
+    return cb
```

Comment:
```suggestion
    const listenerRef = window.addEventListener('message', handleMsgRec)
    const cb = () => {
      window.removeEventListener('message', listenerRef)
    }
    return cb
```

Also consider `handleMsgRec` being memoized via `useCallback` or `useMemo` to ensure that the window event listener always has the same/correct reference to the function (since currently the function is being re-generated on every re-render, this is especially important if that function depends on any state e.g. `msg`)